app/templates/crates.hbs

```diff
@@ -59,7 +60,7 @@
     {{#link-to 'crates' (query-params page=prevPage) class="prev"}}
         <img src="/assets/left-pag.png"/>
     {{/link-to}}
-    {{#each pages}}
+    {{#each surroundingPages}}
```

Comment:
I think that this should probably be done for all pagination, so you may just want to merge the `surroundingPages` method into `pages`. If we need to we can add an `allPages` method later on.